boards/px4/fmu-v6x/src/spi.cpp

```diff
@@ -84,9 +84,9 @@ constexpr px4_spi_bus_all_hw_t px4_spi_buses_all_hw[BOARD_NUM_SPI_CFG_HW_VERSION
 		}),
 	}),
 
-	initSPIHWVersion(V6X50, {
+	initSPIHWVersion(V6X43, {
```

Comment:
Add V6X43 and keep V6X50

Reply:
Ok, that's plus around 700 B of flash, unfortunately.

And note, this PR is just the backport to v1.13.